docs/plugins/types.md

```diff
@@ -41,6 +59,22 @@ Parent directory.
 | ---- | ------- |
 | Type | `Self?` |
 
+### `base` {#url.base}
```

Comment:
The definition of the URL "base" isn't settled yet and may change in the future, it's safer to leave it undocumented for now